components/Button.html

```diff
@@ -36,7 +36,91 @@
+                    <p class="component__text">Buttons allow users to take actions, and make choices,
+                         with a single tap. Use different classes created for different button 
+                         styles to implement in your project.</p>
+                    <p class="component__heading-primary">Primary Buttons</p>
+                    <p class="component__sub-text">
+                         Here are some Primary style buttons you can use by integrating classes for 
+                         differnt styles like for <span class="comp__detail-text">primary button</span class="comp__detail-text"> use <span class="comp__detail-text">.solid__primary</span> 
+                         along with <span class="comp__detail-text">.btn</span> class.
+                    </p> 
+                    <div class="component__view avatar__comp">
+                        <button class="btn solid__primary">Primary Button</button>
+                        <button class="btn ouline__btn">Outline Button</button>
+                        <button class="btn solid__primary icon__btn"><i class='bx bxs-cart-alt'></i>Go to cart </button>
+                        <button class="btn link__btn"><a href="#" class="btn-link">Primary-Link-btn</a></button>
+                        <button class="btn btn__icon-primary"><i class='bx bxs-home-smile'></i></button>
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                            src="https://carbon.now.sh/embed/OVhp6YhPTvi9BoxpMrff"
+                            style="width: 960px; height: 276px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+
+                    <p class="component__heading-primary">Secondry Button Styles</p>
+                    <p class="component__sub-text">
+                        Here are some Secondry style buttons you can use by integrating classes for 
+                        differnt styles like for <span class="comp__detail-text">secondry button</span class="comp__detail-text"> use <span class="comp__detail-text">.solid__secondry</span> 
+                        along with <span class="comp__detail-text">.btn</span> class.
+                    </p> 
+                    <div class="component__view avatar__comp">
+                        <button class="btn solid__secondry">Primary Button Styles</button>
+                        <button class="btn secondry__ouline-btn">Outline Button</button>
+                        <button class="btn solid__secondry icon__btn"><i class='bx bx-left-arrow-alt'></i>Go Back </button>
+                        <button class="btn secondry__link-btn"><a href="#" class="secondry__link">Primary-Link-btn</a></button>
+                        <button class="btn btn__icon-secondry"><i class='bx bx-image-add'></i></button>
```

Comment:
dont mix them as you are using "solid__secondary" follow that method only and do "outline__secondary" instead of "secondary__outline-btn" as this will cause friction while developing